geomesa-datastore-bundle/geomesa-datastore-processors/pom.xml

```diff
@@ -66,6 +66,10 @@
             <groupId>org.geotools.jdbc</groupId>
             <artifactId>gt-jdbc-postgis</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-tools_2.11</artifactId>
```

Comment:
Ok, why the added dependency on geomesa-tools?

Reply:
this is to pull in the exporters for the ConvertToGeoFile so we can output more than just avro!